core/templates/dev/head/pages/exploration_editor/editor_tab/InteractionDetailsCacheServiceSpec.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for Interaction Details Cache Service.
+ */
+describe('Interaction Details Cache Service', function() {
+  describe('InteractionDetailsCache', function() {
+    beforeEach(function() {
+      module('oppia');
+    });
+
+
+    var customizationArgs = {
```

Comment:
Let's make it `interactionCustomizationArgs` and also it should contain a key that an actual object will contain, for example you can make it
```
var interactionCustomizationArgs = {
      choices: {
          value: 'SampleChoice'
      }
}
```